rcl/src/rcl/logging.c

```diff
@@ -201,6 +201,37 @@ rcl_logging_ext_lib_output_handler(
   }
 }
 
+bool
+rcl_contains_logging_arguments(int argc, char const * const * argv)
```

Comment:
@ivanpauno I'm not particularly fond of coupling logging and arguments this much. But I'm a bit divided, as I can imagine why you chose keeping this function here. @wjwwood thoughts?

Reply:
Well, I think that moving to `arguments.c` makes sense.

Reply:
I don't have a strong opinion.